storage/entity.go

```diff
@@ -0,0 +1,350 @@
+package storage
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"io/ioutil"
+
+	"github.com/satori/uuid"
+)
+
+// Annotating as secure for gas scanning
+/* #nosec */
+const (
+	partitionKeyNode  = "PartitionKey"
+	rowKeyNode        = "RowKey"
+	etagErrorTemplate = "Etag didn't match: %v"
+)
+
+// Entity represents an entity inside an Azure table.
+type Entity struct {
+	tsc           *TableServiceClient
```

Comment:
Why there is a need to link each entity to table service client and the table? Tables can have millions of rows.
Entity is just data. What if I want to insert the same data into two tables in two different storage accounts?